### ISSUE TITLE
Easylogging config option %time doesn't work

### ISSUE THREAD

**Issue by andysCaplin** [hazelcast/hazelcast-cpp-client#585]
C++ compiler version: N/A
Hazelcast Cpp client version: 3.12
Hazelcast server version: 3.12
Number of the clients: N/A
Cluster size, i.e. the number of Hazelcast cluster members: N/A
OS version (Windows/Linux/OSX): Linux

Please attach relevant logs and files for client and server side.

#### Expected behaviour
Using %time as a logging configuration option in a logging configuration file should cause time to be printed in logs.

#### Actual behaviour
It causes %time to be printed in logs e.g.
```
15/04/2020 %time : (0x7f13eb3c9700): INFO: (20190418:3f465ee6) LifecycleService::LifecycleEvent STARTING
```
The %date, %datetime, %thread options all work. Only %time seems to be broken.

#### Steps to reproduce the behaviour
Use this as the logging configuration file passed to `setConfigurationFileName()`
```
* ALL: // We need to start level with star '*' and end with colon ':'
    FORMAT                              =       %date %time : (%thread): %level: %log  // Quotes are not necessary but to make it more readable we wrap quotes around.
    FILENAME                            =       var/cluster-cloud.log
    ENABLED                                     =       true
    TO_FILE                             =       true
    TO_STANDARD_OUTPUT          =       false
    MILLISECONDS_WIDTH          =       6
    PERFORMANCE_TRACKING        =       false
    ROLL_OUT_SIZE               =  2097152 // Throw log files away after 2MB

* DEBUG:
   ENABLED              = false
```

**Comment by ihsandemir** [comment]
Hello @andysCaplin We utilized the easylogging library (https://github.com/amrayn/easyloggingpp/tree/v8.91) underneath and it processes the config file. It looks like it can not handle `%time` format.

Well, actually we will retire easylogging in the upcoming release and use another logging library that utilizes C++11 capabilities (We could not do this until today since we had to use C++98 compatible, but we are moving to C++11 in this upcoming release, hence there will a lot changes including the logging library).

I am setting the milestone to 4.0 for this issue until the new logger is in place. By the way, I also understand that this is not a blocker since you were able to use the alternative format `%datetime` which is a lot more powerfull.

Thanks for reporting this issue.

**Comment by andysCaplin** [comment]
It is minor, we wanted to use %time because using %datetime means we produce a log with a timestamp that's inconsistent with our other logs but it's not a big deal.

**Comment by ihsandemir** [comment]
fixed by #638 We now have integration point to any logging framework where you the user can have full  control of the logger. You just can configure using the logging configuration. Hence, the easylogging is removed from the library now.
